install/self_managed/00_choose_setup_method.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Step 0: Choose Setup Method'
+install_track_title: Installation Guide (Self-Managed)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+To monitor a new Postgres server with pganalyze, you must make some configuration changes
+to your database and install and run the pganalyze collector. There are two
+ways of doing this: manually stepping through our instructions, or running the guided
```

Comment:
This might work better as a list, for better readability (similar to how we present "Log-based EXPLAIN" vs "auto_explain" as two choices)